draft-ietf-core-dns-over-coap.md

```diff
@@ -419,14 +416,14 @@ security properties, which are compiled in that document.
 
 Mapping DoC to DoH
 ------------------
-This document provides no specification how to map between DoC and DoH, e.g., at a CoAP-HTTP-proxy,
-and it is NOT RECOMMENDED.
+This document provides no specification how to map between DoC and DoH, e.g., at a CoAP-HTTP-proxy.
+In fact, such mapping is NOT RECOMMENDED:
 Rewriting the FETCH method ({{sec:queries}}) and the TTL rewriting ({{sec:resp-caching}}) as
 specified in this draft would be non-trivial.
 It is RECOMMENDED to use a DNS forwarder to map between DoC and DoH, as would be the case for
 mapping between any other pair of DNS transports.
 
-Considerations for Unencrypted Use {#sec:unencrypted-coap}
+Considerations for Unprotected Use {#sec:unprotected-coap}
 ==================================
 The use of DoC without a security mode of CoAP is NOT RECOMMENDED.
```

Comment:
We could be using the term NoSec mode (Section 9 of RFCs  7252).
(This is a "security mode", except that it has "No Security". :-)

Reply:
Mh, reading Section 9 of RFC 7252 again, the wording there makes it very DTLS-specific... And makes me think that arbitrarily picking the term “security mode” as an umbrella term for DTLS and OSCORE was a mistake. We did not use the term elsewhere in this document, so maybe just use “without any encryption”? Because in this particular case we _are_ talking about confidential communication here.

Reply:
without confidentiality protection?

(you can encrypt, but still do it wrong)